visualization/app/codeCharta/ui/attributeTypeSelector/attributeTypeSelector.component.ts

```diff
@@ -17,10 +18,17 @@ export class AttributeTypeSelectorController {
 
 	/* @ngInject */
 	constructor(
+		private $rootScope: IRootScopeService,
 		private storeService: StoreService,
 		private nodeMetricDataService: NodeMetricDataService,
 		private edgeMetricDataService: EdgeMetricDataService
-	) {}
+	) {
+		this.$rootScope.$on(StoreService.STORE_CHANGED_EVENT, (_event_, data) => {
+			if (data.actionType === AttributeTypesActions.UPDATE_ATTRIBUTE_TYPE) {
+				this.setAggregationSymbol()
+			}
+		})
+	}
```

Comment:
Could you use a redux property for this?
I would rather avoid listening to all store calls in a specific component and use the observer pattern.
The store should notify a subscriber that a relevant change has occured.